templates/zerver/integrations/jira-plugin.md

```diff
@@ -0,0 +1,50 @@
+_If you are running JIRA version 5.2 or greater, or using the hosted JIRA provided by Atlassian, we recommend using
+the [web-hook method](#jira) above instead. This plugin supports older versions of JIRA._
+
+First, create the stream you'd like to use for JIRA notifications, and subscribe all interested parties to this
+stream. We recommend the name `jira`.
+
+##### Plugin mechanism
+
+Next, download and install our [Python bindings and example scripts](/api).
+
+**Plugin Installation**
+
+The JIRA integration plugin requires two JIRA plugins. Please install the following plugins using the Universal
+Plugin Manager in your JIRA installation:
+
+*   [Script Runner Plugin](https://marketplace.atlassian.com/plugins/com.onresolve.jira.groovy.groovyrunner)
+*   [SSL Plugin](https://marketplace.atlassian.com/plugins/com.atlassian.jira.plugin.jirasslplugin)
+
+**SSL Setup**
+
+As Zulip is using a StartCOM SSL certificate that is not recognized by default in the Java installation shipped with
+JIRA, you will need to tell JIRA about the certificate. Navigate to `Administration > System > Configure SSL` and in
+the `Import SSL Certificates` field, enter `{{ external_api_uri_subdomain }}`. After clicking `Save Certificates`,
+follow the on-screen instructions and restart JIRA for it to recognize the proper certificates.
+
+**Zulip Integration**
+
+Copy the folder `integrations/jira/org/` (from the tarball you downloaded above) to your JIRA `classes` folder. For
+self-contained JIRA installations, this will be `atlassian-jira/WEB-INF/classes/`, but this may be different in your
+deployment. Edit the constants at the top of `org/zulip/jira/ZulipListener.groovy` and fill them with the
+appropriate values:
+
+```Python
+String zulipEmail = "jira-notifications-bot@example.com"
+String zulipAPIKey = "0123456789abcdef0123456789abcdef"
+String zulipStream = "JIRA"
+String issueBaseUrl = "https://jira.COMPANY.com/browse/"
+```
+
+In the Administrators page, navigate to `Plugins > Other > Script Listeners`. In the `Add Listener` section, click
+on the `Custom Listener` option. Select the events you wish the Zulip integration to fire for, and the projects you
+wish Zulip to be notified for. In the `Name of groovy class` field, enter `org.zulip.jira.ZulipListener`.
+
+Click `Add Listener`, and JIRA will now notify your Zulip of changes to your issues!
+
+**Congratulations! You're done!**
+
+Updates from JIRA will be sent to the stream you've configured, as below.
+
```

Comment:
Here again, you can move line 48 to after line 44 and then use `congrats.md`. :)